scripts/TestAllDownloads.expected.txt

```diff
@@ -96,12 +96,16 @@ TestAllDownloads/
       }
   5/
     elm-tooling/
+      elm-format/
+        0.8.6/
+          elm-format
```

Comment:
Is there a command to update this file automatically?

Reply:
Yes: https://elm-tooling.github.io/elm-tooling-cli/contributing/#adding-a-new-version

Reply:
:+1: updated my script for next time